app/src/main/java/no/usn/mob3000/data/model/content/DocsDto.kt

```diff
@@ -21,7 +21,7 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class DocsDto(
     @SerialName("id")
-    val docId: String,
+    val docId: String? = null,
```

Comment:
Documentation will always have an ID. This change makes the tabulated data more difficult to work with and needs to be reverted.